{"name": "MSS files are correctly stored and logged", "status": "failed", "statusDetails": {"message": "AssertionError: \nExpected: an object with length of <1>\n     but: was <[]> with length of <0>\n\n", "trace": "  File \"C:\\Python\\conda\\envs\\demo\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python\\conda\\envs\\demo\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"..\\..\\aviation-data-services\\aviation-data-processing-service\\service_tests\\aviation_data_processing\\features\\steps\\AviationDataProcessing.py\", line 35, in then_it_is_logged\n    assert_that(captured_logs, has_length(equal_to(1)))\n  File \"C:\\Python\\conda\\envs\\demo\\lib\\site-packages\\hamcrest\\core\\assert_that.py\", line 58, in assert_that\n    _assert_match(actual=actual, matcher=matcher, reason=reason)\n  File \"C:\\Python\\conda\\envs\\demo\\lib\\site-packages\\hamcrest\\core\\assert_that.py\", line 73, in _assert_match\n    raise AssertionError(description)\n"}, "steps": [{"name": "Given an icing_mlevs file needs to be processed by icingfilecreatedlambda", "status": "passed", "start": 1624528842305, "stop": 1624528842306}, {"name": "And it has arrived", "status": "passed", "start": 1624528842306, "stop": 1624528843279}, {"name": "When it is processed for 30 seconds", "status": "passed", "start": 1624528843280, "stop": 1624528873284}, {"name": "Then it is logged with [REPORT]", "status": "failed", "statusDetails": {"message": "AssertionError: \nExpected: an object with length of <1>\n     but: was <[]> with length of <0>\n\n", "trace": "  File \"C:\\Python\\conda\\envs\\demo\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python\\conda\\envs\\demo\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"..\\..\\aviation-data-services\\aviation-data-processing-service\\service_tests\\aviation_data_processing\\features\\steps\\AviationDataProcessing.py\", line 35, in then_it_is_logged\n    assert_that(captured_logs, has_length(equal_to(1)))\n  File \"C:\\Python\\conda\\envs\\demo\\lib\\site-packages\\hamcrest\\core\\assert_that.py\", line 58, in assert_that\n    _assert_match(actual=actual, matcher=matcher, reason=reason)\n  File \"C:\\Python\\conda\\envs\\demo\\lib\\site-packages\\hamcrest\\core\\assert_that.py\", line 73, in _assert_match\n    raise AssertionError(description)\n"}, "start": 1624528873285, "stop": 1624528873990}], "start": 1624528842304, "stop": 1624528873991, "uuid": "c89a167b-1e60-4b9e-9026-9f163785ea9e", "historyId": "384d3430391fac3cf5530c25ce250fe1", "fullName": "Scenarios to test aviation data processing: MSS files are correctly stored and logged", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Scenarios to test aviation data processing"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}